### PR TITLE
[Agent App SDK] Documentation for Fullscreen widgets

### PR DESCRIPTION
**Pull request by bolchowka**
Docs for Fullscreen widgets:
* `createFullscreenWidget()`
* `widget.setNotificationBadge()`

### CODE REVIEW

File: content/extending-ui/extending-agent-app/agent-app-sdk/index.mdx [hunk 1]
```diff
@@ -339,6 +340,57 @@ widget.putMessage(richMessage);
   Read more about Rich Messages
 </SectionLink>
 
+## Fullscreen widgets
+
+If you want to connect the Fullscreen widget to the LiveChat Agent App, you should use the `createFullscreenWidget` function. It returns a promise resolving to a fullscreen widget instance.
```
Comment (ale-kacperczyk) [inline content/extending-ui/extending-agent-app/agent-app-sdk/index.mdx]:
```suggestion
If you want to connect a Fullscreen widget to the LiveChat Agent App, you should use the `createFullscreenWidget` function. It returns a promise resolving to a Fullscreen widget instance.
```

File: content/extending-ui/extending-agent-app/index.mdx [hunk 2]
```diff
@@ -85,4 +85,4 @@ Some applications require settings configuration, especially when they use exter
 
 We provide a special place called Settings page, where you can embed a page with configuration. It's recommended to use it together with [Sign in with LiveChat](/getting-started/authorization/sign-in-with-livechat/).
 
-See how to set up a Settings page for your app in the [documentation](/extending-ui/extending-agent-app/agent-app-sdk/#getting-started).
```
Comment (ale-kacperczyk) [inline content/extending-ui/extending-agent-app/index.mdx]: I'm not sure why this link was changed 🤔

File: content/extending-ui/extending-agent-app/agent-app-sdk/index.mdx [hunk 3]
```diff
@@ -339,6 +340,57 @@ widget.putMessage(richMessage);
   Read more about Rich Messages
 </SectionLink>
 
+## Fullscreen widgets
```
Comment (ale-kacperczyk) [inline content/extending-ui/extending-agent-app/agent-app-sdk/index.mdx]: We call them Fullscreen apps throughout the documentation so maybe we should stick to this name.

File: content/extending-ui/extending-agent-app/index.mdx (hunk 2 above)
Comment (bolchowka) [reply content/extending-ui/extending-agent-app/index.mdx]: Good catch, I made a mistake and replaced another link, not the one I intended. Fixed in e613edb

File: content/extending-ui/extending-agent-app/agent-app-sdk/index.mdx (hunk 3 above)
Comment (fjaskolski) [reply content/extending-ui/extending-agent-app/agent-app-sdk/index.mdx]: I second this! 🙌

**Pull request merged**